core/controllers/practice_sessions.py

```diff
@@ -32,9 +32,6 @@ def get(self, topic_name):
 
         topic = topic_services.get_topic_by_name(topic_name)
 
-        if topic is None:
```

Comment:
@lilithxxx, is there any specific reason for deleting this line?

Reply:
This condition is already checked on `acl_decorators.can_access_topic_viewer_page()` [here](https://github.com/oppia/oppia/blob/develop/core/controllers/acl_decorators.py#L2344). I removed this line as this condition can never be reached since the Exception would be rendered from the acl_decorators file.